Client/Layout/MainLayout.razor

```diff
@@ -2,15 +2,41 @@
 
 <RadzenComponents />
 <div class="page">
-    <div class="sidebar">
-        <NavMenu />
-    </div>
-
     <main>
-        <div class = "app_bar">
-            <AppBar/>
+        <div class="header">
+            <h1 class="title">
+                <img src="Logo.png" alt="logo" class="logo" />
+                Contrasting Horizons</h1>
+            <nav>
+                <ul class="nav_links">
+                    <li><NavLink href="/">Home</NavLink></li>
+                    <li><NavLink href="/Compare">Compare</NavLink></li>
+                    <li><NavLink href="/about">About</NavLink></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="search-container">
+            <input type="text" placeholder="Search for a country..." name="search" />
+            <button type="submit">Search</button>            
         </div>
 
+        <div>
+            <p class="paragraph_country">
+                <img src="sweden-flag.png" alt="Sweden" class="flag" />
+                Sweden</p>
+        </div>
+
+        <div class="grid-container">
```

Comment:
Shouldn't this be a part of the home page instead of the main layout? The main layout is applied to all pages and should probably only consist of the nav bar and other elements that are present on all pages. Do we really want to have this grid on all pages that we could have on the website?

Reply:
We dont have a home page thats why its there but when the home page is created the code can be moved to home page.

Reply:
We do have a home page _(Pages/Home.razor)_.
Also, if we want to place any data in the grid, the grid must be **inside** the DashBoard component.

Reply:
Fixed it. Now everything that doesn't belongs to the MainLayout is moved to Home page.